src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleFastExp.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.player
+
+import net.ccbluex.liquidbounce.config.ToggleableConfigurable
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.features.module.modules.player.invcleaner.HotbarItemSlot
+import net.ccbluex.liquidbounce.features.module.modules.player.invcleaner.OffHandSlot
+import net.ccbluex.liquidbounce.utils.aiming.Rotation
+import net.ccbluex.liquidbounce.utils.aiming.RotationManager
+import net.ccbluex.liquidbounce.utils.aiming.RotationsConfigurable
+import net.ccbluex.liquidbounce.utils.combat.CombatManager
+import net.ccbluex.liquidbounce.utils.input.InputBind
+import net.ccbluex.liquidbounce.utils.inventory.InventoryManager
+import net.ccbluex.liquidbounce.utils.inventory.OFFHAND_SLOT
+import net.ccbluex.liquidbounce.utils.inventory.useHotbarSlotOrOffhand
+import net.ccbluex.liquidbounce.utils.item.findHotbarItemSlot
+import net.ccbluex.liquidbounce.utils.item.getEnchantment
+import net.ccbluex.liquidbounce.utils.kotlin.Priority
+import net.minecraft.enchantment.Enchantments
+import net.minecraft.item.ItemStack
+import net.minecraft.item.Items
+
+/**
+ * FastExp module
+ *
+ * Automatically repairs your armor.
+ */
+object ModuleFastExp : Module(
+    "FastExp",
+    Category.PLAYER,
+    bindAction = InputBind.BindAction.HOLD,
+    disableOnQuit = true
+) {
+
+    private object Rotate : ToggleableConfigurable(this, "Rotate", true) {
+        val rotations = tree(RotationsConfigurable(this))
+    }
+
+    init {
+        tree(Rotate)
+    }
+
+    private val noWaste by boolean("NoWaste", true)
+    private val itemsPerTick by int("ItemsPerTick", 5, 1..32)
+    private val combatPauseTime by int("CombatPauseTime", 0, 0..40, "ticks")
+    private val slotResetDelay by intRange("SlotResetDelay", 0..0, 0..40, "ticks")
+
+    override fun disable() {
+        mc.options.useKey.isPressed = false
```

Comment:
Check if use key is being used before setting it to false

Reply:
Mh you did it like that in AutoBuff so i though it's fine. I just remove that line as it doesn't need it anyways.